install/autodetect.go

```diff
@@ -84,7 +84,6 @@ func (k *K8sInstaller) detectDatapathMode(withKPR bool) error {
 		k.params.DatapathMode = DatapathTunnel
 
 		if withKPR && k.params.KubeProxyReplacement == "" {
-			k.Log("ℹ️  kube-proxy-replacement disabled")
```

Comment:
Should we remove the same log message for AKS [on line 110](https://github.com/cilium/cilium-cli/pull/1918/files#diff-70c8da281ad8db17039130d2e7ad09aa2fc7db3f6d44217e6db0764bef731fffR110) as well?

Reply:
yeah let's do that 👍 i'll update the PR today